<ui version="4.0" >
 <class>relabelForm</class>
 <widget class="QDialog" name="relabelForm" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Label</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QLabel" name="fromLabel" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>From Volume :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>382</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="3" column="1" >
      <widget class="QSpinBox" name="slotSpin" >
       <property name="maximum" >
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Pool:</string>
       </property>
       <property name="buddy" >
        <cstring>poolCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Storage:</string>
       </property>
       <property name="buddy" >
        <cstring>storageCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="volumeName" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="poolCombo" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>New Volume Name:</string>
       </property>
       <property name="buddy" >
        <cstring>volumeName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="storageCombo" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Slot:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>71</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Relabel a Volume&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>81</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>relabelForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>relabelForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
